cmd/util/cmd/rollback-executed-height/cmd/rollback_executed_height_test.go

```diff
@@ -218,7 +243,10 @@ func TestReExecuteBlockWithDifferentResult(t *testing.T) {
 			header.ID(),
 		)
 
+		err2 := batch.Flush()
```

Comment:
same comment here, better to check remove and flush twice instead of remove twice and flush once.